### PR TITLE
Lesson 4

### PR DESCRIPTION
**Pull request by viktoriiaso**

### CODE REVIEW

File: lesson-3/3lesson_dz.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# 1. Використовуючи функцію print, надрукувати фразу “I love Python” 42 рази
```
Comment (klymenok) [inline lesson-3/3lesson_dz.py]: This file shouldn't be in this PR. Remove it, please, after you merge your previous PR

File: lesson-4/4lesson_dz.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#1. Створити програму, яка буде очікувати введення тексту від користувача
+# і повідомляти — чи є введений текст “числом” чи “словом”.
+#2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+#3. Якщо це “слово”, програма має вказати його довжину
+
+some_text = input('Enter some text')
+print(type(some_text))
+
+if some_text.isdigit():
+    num = int(some_text)%2
```
Comment (klymenok) [inline lesson-4/4lesson_dz.py]: It is not necessary to create a variable here. You can use `int(some_text) % 2` directly in condition: `if int(some_text) % 2:`

**Review by klymenok — CHANGES_REQUESTED** [review]
Looks good, but there is a file that shouldn't be here. Make updates, please

File: lesson-3/3lesson_dz.py (hunk 1 above)
Comment (viktoriiaso) [reply lesson-3/3lesson_dz.py]: files removed

File: lesson-4/4lesson_dz.py (hunk 2 above)
Comment (viktoriiaso) [reply lesson-4/4lesson_dz.py]: done

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**